ex01.py

```diff
@@ -0,0 +1,16 @@
+def flatten(x):
```

Comment:
zmienne nazywajmy tak, żeby to miało sens, teraz to nie wiesz czym jest ten x, ogólnie jednoliterowe nazwy to wg mnie powinna być rzadkość

Reply:
zmieniono nazwy